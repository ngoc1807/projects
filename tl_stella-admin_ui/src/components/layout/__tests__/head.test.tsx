import { cleanup, render } from "@testing-library/react";
import { Head } from "../head";

describe("head", () => {
  afterEach(() => {
    cleanup();
  });
  it("should render correctly", () => {
    const { container } = render(<Head />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
