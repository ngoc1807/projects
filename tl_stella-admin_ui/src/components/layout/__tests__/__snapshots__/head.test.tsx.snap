// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`head should render correctly 1`] = `null`;
